src/deleteToList.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-cycle
+import { stock, displayToDoList } from './index.js';
+
+export default (e) => {
+  stock = stock.filter((x) => x.index !== Number(e.target.parentElement.id - 1));
+  for (let i = 0; i < stock.length; i += 1) {
+    stock[i].index = i + 1;
+  }
+  displayToDoList(stock);
+  localStorage.setItem('stock', JSON.stringify(stock));
+};
```

Comment:
- [ ] It seems there is a bug when a user tries to delete a single task, all tasks are removed from the screen and the task is not removed from the localStorage. Let's try to improve this behavior as it's not expected.

![todo_list_no_delete](https://user-images.githubusercontent.com/79114233/144711898-3fdc8848-100f-42e9-ae5f-fce4297152b8.gif)